cpp/src/arrow/compute/exec/hash_join_node_test.cc

```diff
@@ -1061,16 +1061,18 @@ TEST(HashJoin, Random) {
 
     // Print test case parameters
     // print num_rows, batch_size, join_type, join_cmp
-    std::cout << join_type_name << " " << key_cmp_str << " ";
-    key_types.Print();
-    std::cout << " payload_l: ";
-    payload_types[0].Print();
-    std::cout << " payload_r: ";
-    payload_types[1].Print();
-    std::cout << " num_rows_l = " << num_rows_l << " num_rows_r = " << num_rows_r
-              << " batch size = " << batch_size
-              << " parallel = " << (parallel ? "true" : "false");
-    std::cout << std::endl;
+    // std::cout << join_type_name << " " << key_cmp_str << " ";
+    // key_types.Print();
+    // std::cout << " payload_l: ";
+    // payload_types[0].Print();
+    // std::cout << " payload_r: ";
+    // payload_types[1].Print();
+    // std::cout << " num_rows_l = " << num_rows_l << " num_rows_r = " << num_rows_r
+    //           << " batch size = " << batch_size
+    //           << " parallel = " << (parallel ? "true" : "false");
+    // std::cout << std::endl;
```

Comment:
Go ahead and just delete these lines entirely.  No need to leave the commented out code.